homeassistant/components/homematicip_cloud/services.py

```diff
@@ -42,6 +43,7 @@
 SERVICE_DUMP_HAP_CONFIG = "dump_hap_config"
 SERVICE_RESET_ENERGY_COUNTER = "reset_energy_counter"
 SERVICE_SET_ACTIVE_CLIMATE_PROFILE = "set_active_climate_profile"
+SERVICE_SET_COOLING = "set_cooling"
```

Comment:
Should we call it `set_home_cooling_mode` or something to make it a bit more specific what it is (+ update the wording in `strings.json`)?

Reply:
The question is more related to what's it called in the app so the user "recognizes" what this service is.